app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestsFragment.java

```diff
@@ -24,6 +33,43 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState)
     {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_requests, container, false);
+        view = inflater.inflate(R.layout.fragment_requests, container, false);
```

Comment:
Read the comment below, should this be camel case instead? (requestsFragment instead of fragment_requests)?

Reply:
I think everything xml uses underscores. That's referencing the layout not the class

Reply:
You are right. This is actually an error I need to fix with the login screen and create account screen